LICENSE

```diff
@@ -1,8 +1,11 @@
-MIT License
+# MIT License
```

Comment:
This isn't a markdown file

Reply:
resolved on c9ffd78ddc93ede49af145fe706ee269c4e0a42c